base_rest_pydantic/i18n/base_rest_datamodel.pot

```diff
@@ -0,0 +1,26 @@
+# Translation of Odoo Server.
```

Comment:
I don't know if this will break anything but this should be named `base_rest_pydantic.pot`